Formula/ship.rb

```diff
@@ -31,7 +31,7 @@ def install
     assert_match(/#{version}/, shell_output("#{bin}/ship version"))
     assert_match(/Usage:/, shell_output("#{bin}/ship --help"))
 
-    test_chart = "https://github.com/replicatedhq/test-charts/tree/master/plain-k8s"
+    test_chart = "https://github.com/replicatedhq/test-charts/tree/main/plain-k8s"
```

Comment:
```suggestion
    test_chart = "https://github.com/replicatedhq/test-charts/tree/HEAD/plain-k8s"
```
This might also work so that we don't have to worry about the name of the default branch.

Reply:
💯 👉🏼 https://github.com/Homebrew/homebrew-core/pull/91489